README.md

```
@@ -1,87 +1,188 @@
+<div align="center">
 
+```
+               __                     __    __        __                       __
+ _  __ ____ __/ /____  ___ _______ _/ /   / /_____ _/ /____ ____ _____ ____  / /_
+| |/ // / // / // _ \/ -_) __/ _ `/ _ \ / __/ _ `/ __/ _ `/ -_) __/ _ `/ -_)/ __/
+|___/\_,_/_//_//_//_/\__/_/  \_,_/_.__/ \__/\_,_/_/  \_, /\__/\__/\_, /\__/ \__/
+                                                    /___/        /___/
+```
 
-<img width="530" height="137" alt="image" src="https://github.com/user-attachments/assets/5601f3a8-7037-4fe5-8115-eb4182d8ab2e" />
+**Create intentionally vulnerable environments for security testing, education, and research**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)